ComponentAssemblies/ClientComponents/StatisticsWidget/projects/StatisticsWidget.json

```
@@ -0,0 +1,147 @@
+      "fraction" : 1,
+      "type" : "R[6]"
+    },
+    "type" : "dev",
+    "v" : 6,
+    "viewUUID" : "MAINVIEW"
+  },
+  "partitions" : [ ],
+  "resources" : [ {
+    "label" : "com.vantiq.appcomponents.MichaelAC",
+    "name" : "com.vantiq.appcomponents.MichaelAC",
+    "resourceReference" : "/collaborationtypes/com.vantiq.appcomponents.MichaelAC",
+    "timestamp" : "2025-02-26T23:58:41.227Z",
+    "type" : 12
+  }, {
+    "label" : "com.vantiq.clientcomponent.Statistics",
+    "name" : "com.vantiq.clientcomponent.Statistics",
+    "resourceReference" : "/system.clients/com.vantiq.clientcomponent.Statistics",
+    "timestamp" : "2025-04-07T17:07:43.478Z",
+    "type" : 72
+  }, {
+    "label" : "com.vantiq.themes.Themes",
+    "name" : "com.vantiq.themes.Themes",
+    "resourceReference" : "/system.clients/com.vantiq.themes.Themes",
+    "timestamp" : "2025-03-24T17:47:11.890Z",
+    "type" : 15
+  }, {
+    "label" : "vantiq/themes/Cornflower.thm",
+    "name" : "vantiq/themes/Cornflower.thm",
+    "resourceReference" : "/documents/vantiq/themes/Cornflower.thm",
+    "timestamp" : "2025-03-24T17:54:41.917Z",
+    "type" : 19
+  } ],
+  "selectData" : { },
+  "tools" : [ {
+    "isPinned" : false,
+    "name" : "Catalogs",
+    "pane" : {
+      "address" : "R[4]",
+      "isActive" : true,
+      "isActiveTool" : true
+    },
+    "state" : 8,
+    "type" : 115
+  }, {
+    "isPinned" : false,
+    "name" : "Documents",
+    "pane" : {
+      "address" : "R[3]"
+    },
+    "state" : 8,
+    "type" : 26
+  }, {
+    "isPinned" : false,
+    "name" : "Errors",
+    "type" : 13
+  }, {
+    "dockLocation" : "top",
+    "isPinned" : false,
+    "name" : "Inactive Panes",
+    "type" : 99
+  }, {
+    "isPinned" : false,
+    "name" : "Manage Catalog",
+    "pane" : {
+      "address" : "R[5]"
+    },
+    "state" : 8,
+    "type" : 114
+  }, {
+    "isPinned" : false,
+    "name" : "Project Contents",
+    "type" : 2
+  }, {
+    "isPinned" : false,
+    "name" : "StatisticsWidget",
+    "pane" : {
+      "address" : "R[1]"
+    },
+    "state" : 8,
+    "type" : 120
+  }, {
+    "isPinned" : false,
+    "name" : "com.vantiq.clientcomponent.Statistics",
+    "pane" : {
+      "address" : "R[0]"
+    },
+    "resourceKey" : "client/com.vantiq.clientcomponent.Statistics",
+    "state" : 8,
+    "toolOptions" : {
+      "isRunning" : false
+    },
+    "type" : 63
+  }, {
+    "isPinned" : false,
+    "name" : "com.vantiq.themes.Themes",
+    "pane" : {
+      "address" : "R[2]"
+    },
+    "resourceKey" : "client/com.vantiq.themes.Themes",
+    "state" : 8,
+    "type" : 63
+  } ],
+  "type" : "dev",
+  "views" : [ {
+    "name" : "Project Contents",
+    "projectToolKeys" : [ "catalogs/Catalogs", "documents/Documents", "errorviewer/Errors", "tiledock/Inactive Panes", "managecatalog/Manage Catalog", "list/Project Contents", "assemblies/StatisticsWidget", "client/com.vantiq.clientcomponent.Statistics", "client/com.vantiq.themes.Themes" ]
```

remove Theme here, too
And probably Manage Catalog pane   (here and further above)
```suggestion
    "projectToolKeys" : [  "documents/Documents", "errorviewer/Errors", "tiledock/Inactive Panes", "list/Project Contents", "assemblies/StatisticsWidget", "client/com.vantiq.clientcomponent.Statistics" ]
```